solarforecastarbiter/reference_forecasts/main.py

```diff
@@ -465,3 +465,143 @@ def make_latest_nwp_forecasts(token, run_time, issue_buffer, base_url=None):
         logger.info('No forecasts to be made at %s', run_time)
         return
     process_nwp_forecast_groups(session, run_time, execute_for)
+
+
+def _is_reference_persistence_forecast(extra_params_string):
+    match = re.search('is_reference_persistence_forecast(["\\s\\:]*)true',
+                      extra_params_string, re.I)
+    return match is not None
+
+
+def generate_reference_persistence_forecast_parameters(
+        session, forecasts, observations, max_run_time):
+    """Sort through all *forecasts* to find those that should be
+    generated by the Arbiter from persisting Observation values. The
+    forecast must have a is_reference_persistence_forecast=true and an
```

Comment:
if it's supposed to be a JSON string then I'd expect `'is_reference_persistence_forecast': true`